docs: Integrate content of landing page (#618)
eclipse-score/score: docs/introduction/index.rst

Architecture
------------

The following figures show a draft version of the planned architecture of the core stack. The architecture will be refined further as part of the project.

High Level View on the Core Stack
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The following high level architecture picture gives you an overview of the main platform building blocks
including hardware specific peripherals.

.. image:: _assets/score_architecture_high_level_overview.png
   :width: 900
   :alt: High level view on the core stack
   :align: center

Target Picture of Core Stack - PRELIMINARY
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The following image shows platform architecture in more details including surroundings as  
Board support package (BSP) and HW specific.

.. image:: _assets/score_architecture_core_stack_target_picture.png
   :width: 900
   :alt: Target picture the core stack (preliminary)
   :align: center

.. _introduction_roadmap:

Roadmap
-------

Here you can find the preliminary roadmap of the project:

.. image:: _assets/score_roadmap.svg
   :alt: project roadmap
   :align: center

> Please be aware, that this roadmap will be also transfered to the `GitHub project <https://github.com/orgs/eclipse-score/projects/1>`_.

Reviewer: ```suggestion
Please be aware, that the roadmap is transferred to and maintained in the `GitHub project <https://github.com/orgs/eclipse-score/projects/13>`_.
```

A lot of preparation was already done in the background, therefore we are quite optimistic to finish the *MVP Phase*
latest in the beginning of 2025.  

Alignment Phase
^^^^^^^^^^^^^^^

In the *Alignment Phase* the main goal is to align on the feature architecture and requirements of the *SCORE Platform v1.0*.
Additionally it is important to define the roadmap and the order, in which the features should be implemented.

Development Phase
^^^^^^^^^^^^^^^^^

The *Development Phase* is the phase, where the implementation of the main modules of the platform should happen.
The most important milestone here is the *Release v0.5*, that should contain the basic modules, e.g *IPC*.
The *Release v0.5* milestone will show, whether previously defined software development process will work and how big is the acceptance and 
the interest of the automotive community to the project.

Series Stability & Evolution Phase
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

In this phase the project should be in an well established state and accepted by the community. Continious development
of the features is taking place.

How we Work
-----------

Meetings
^^^^^^^^

> The following regular meetings (and corresponding meeting minutes) are held as part of the project:
> 
> - `Project Leader Circle <https://github.com/orgs/eclipse-score/discussions/categories/project-lead-circle>`_
> 
> - `Technical Leader Circle <https://github.com/orgs/eclipse-score/discussions/categories/technical-leader-circle>`_
> 
> The dates will be announced via the score-dev@eclipse.org mailing list.
> 
> We plan to start regular exchange in the scope of the *SCORE* project in December 2024.

Reviewer: ```suggestion
Regular meetings (and corresponding meeting minutes) are held as part of the project:

- `Teams & Circles <https://github.com/orgs/eclipse-score/discussions>`_

Typically teams and circles meet weekly.

A public meeting calendar for the *S-CORE* project is in preparation.
```

Reviewer: See also this commit: https://github.com/eclipse-score/eclipse-score.github.io/pull/18/commits/b93ff191a71df1da8f3366d0a5e232bc904ff96c
